src/instructlab/configuration.py

```diff
@@ -92,6 +92,7 @@ def _reset(self):
         self._cache_home = pd.user_cache_dir
         self._config_dir = pd.user_config_dir
         self._data_dir = pd.user_data_dir
+        self._models_dir = path.join(pd.user_data_dir, STORAGE_DIR_NAMES.MODELS)
```

Comment:
The comment above suggested to me that this was a safer approach for the respective unit test - you can see there I used it for checking the default model name was indeed added.

It may be unnecessary - @RobotSail perhaps you could weigh in here? If it's not beneficial I can definitely take a different approach.

Reply:
@nathan-weinberg The `self._<name>_dir` attributes are meant for us to store the location of the data, config, and cache directories. We do it this way so that we can reset their values during testing with the `._reset` method. 

The attribute you're looking for already exists as `DEFAULTS.MODELS_DIR` in the global `DEFAULTS` object, so you shouldn't need to make this change.

Reply:
Thanks!